<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="tb_docx">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Path to template.docx</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="bt_docx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select path to docx
template</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_txt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Path to names.txt</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="bt_txt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select path to txt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>751</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_gen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>751</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generate certificates</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menubout">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menubout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bt_docx</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bt_docx_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_txt</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bt_txt_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bt_gen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>bt_gen_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionAbout_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>bt_docx_clicked()</slot>
  <slot>bt_txt_clicked()</slot>
  <slot>bt_gen_clicked()</slot>
  <slot>actionAbout_clicked()</slot>
 </slots>
</ui>
